src/account.rs

```diff
@@ -166,7 +170,12 @@ impl Account {
             .body(Full::default())?;
 
         let rsp = self.inner.client.http.request(request).await?;
-        Problem::check::<RenewalInfo>(rsp).await
+        let retry_after = crate::order::retry_after(&rsp)
+            .ok_or(Error::Str("Missing Retry-After header"))?
+            .duration_since(std::time::SystemTime::now())
+            .unwrap_or(Duration::ZERO);
```

Comment:
Please rewrite it more like this:

```rust
let Some(retry_after) = retry_after(&rsp) else {
     return Err(Error::Str("missing Retry-After header"));
};

let delay = retry_after.duration_since(SystemTime::now()).unwrap_or(Duration::ZERO);
```

Import all the paths and keep the error message lowercase.